.gitmodules

```diff
@@ -12,5 +12,5 @@
 	branch = master
 [submodule "zwave-adapter"]
 	path = zwave-adapter
-	url = https://github.com/mozilla-iot/zwave-adapter
-	branch = master
+	url = https://github.com/dhylands/zwave-adapter
```

Comment:
Probably should change this back.

Reply:
Will do. I noticed that failures in package.sh weren't stopping the build, so I wanted to test using bash -e in the package.sh which seems to work.